README.md

```diff
@@ -16,16 +15,11 @@ The Open Neural Network eXchange ([ONNX](http://onnx.ai)) is an open format to r
 #### Read the [Usage](#usage-) section below for more details on the file formats in the ONNX Model Zoo (.onnx, .pb, .npz) and starter Python code for validating your ONNX model using test data.
 
 * [Image Classification](#image_classification)
-* [Semantic Segmentation](#semantic_segmentation)
-* [Object Detection & Segmentation](#object_detection)
-* [Face Detection and Recognition](#face_detection)
-* [Emotion Recognition](#emotion_detection)
-* [Gender Detection](#gender_detection)
-* [Hand Written Digit Recognition](#digit_recognition)
-* [Super Resolution](#super_resolution)
+* [Object Detection & Image Segmentation](#object_detection)
+* [Body, Face & Gesture Analysis](#body_analysis)
 * [Style Transfer](#style_transfer)
```

Comment:
Are there a lot of style transfer models? If not maybe have a "image manipulation" category that includes Style transfer and Super resolution.